drivers/net/Kconfig

```diff
@@ -72,6 +78,16 @@ config PPP_CLIENT_CLIENTSERVER
 	  This is only necessary if a ppp connection should be
 	  established with a Microsoft Windows PC.
 
+config PPP_DIALUP
+	bool "Features that are needed if using PPP for a dial up"
+	default n
```

Comment:
The options are off by default so we do not have these `default n` settings in Kconfig files so this line can be removed.